snowflake/repository.go

```diff
@@ -82,9 +97,7 @@ func (repo *SnowflakeRepository) isProtectedRoleName(rn string) bool {
 }
 
 func (repo *SnowflakeRepository) GetDataUsage(ctx context.Context, minTime time.Time, maxTime *time.Time) <-chan stream.MaybeError[UsageQueryResult] {
-	batchSize := 20000
-
-	outputChannel := make(chan stream.MaybeError[UsageQueryResult], batchSize)
+	outputChannel := make(chan stream.MaybeError[UsageQueryResult], 10000)
```

Comment:
Just wondering the `10.000` static value is no issue here?

Reply:
This is 'just' the buffer size for the channel. Writing will block if the buffer is full, but that's not a problem (+ will probably never happen).